### ISSUE TITLE
Issue with files named `common.j` and `blizzard.j`

### ISSUE THREAD

**Issue by nvs** [wurstscript/WurstScript#569]
Hi. I was checking out Wurst (`1.7.0.0-jenkins-Wurst-485`) and ran into the following issue. I was passing numerous Jass files, some of which were named `common.j`, but were located in different directories. Probably not the typical naming scheme, but not one I would expect to fail.

It seems that Wurst properly handles the multiple `common.j` up to a certain point. However, what is put into the `temp/output.j` fails the PJass check as the contents of the files named `common.j` are missing.

This issue is also exhibited with files named `blizzard.j`.

## Example

- File: `src/common.j`

	```
	function A takes nothing returns nothing
	endfunction
	```

- File: `src/B/common.j`

	```
	function B takes nothing returns nothing
	endfunction
	```

- File: `src/main.j`

	```
	function main takes nothing returns nothing
		call A ()
		call B ()
	endfunction
	```
- Execute: `java -jar ~/.wurst/wurstscript.jar src/common.j src/B/common.j src/main.j`

	```
	compilation finished (errors: 2, warnings: 0)
	Error in File output.j line 9:
	 This is a bug in the Wurst Compiler. Please Report it. Pjass has found the following problem:  Undeclared function A
	Error in File output.j line 10:
	 This is a bug in the Wurst Compiler. Please Report it. Pjass has found the following problem:  Undeclared function B
	```

- Execute: `java -jar ~/.wurst/wurstscript.jar src/main.j`
	
	```
	compilation finished (errors: 2, warnings: 0)
	Error in File main.j line 2:
	 Reference to function A could not be resolved.
	Error in File main.j line 3:
	 Reference to function B could not be resolved.
	```

**Comment by peq** [comment]
The `common.j` and `blizzard.j`files are not included in the output, since Wurst assumes that they are already provided by Warcraft. Just use different file names if you want the code to be in the output.

You can also disable the Pjass checks with the `-noPJass` option.

**Comment by nvs** [comment]
Yeah, I figured as much but wasn't sure. Still, I'd rather leave the PJass check on for sanity's sake.

I've run into a few other minor issues when using only Jass files:

- The `debug` keyword (which I don't really make use of) is removed by Wurst. In-game, however, these modified statements are simply ignored. I would have expected the statements to be removed if anything. Easy enough to avoid, however.
- Handles that are already set to `null` are to set `null` again by Wurst. Not really a big deal, but does introduce extra lines to the compiled code.  Still nice to know Wurst has my back.

For now, I'll still be using only Jass files. But I'm looking forward to trying more of what Wurst offers.

**Comment by nvs** [comment]
> Wurst should remove that duplicate statement, and more. Did you pass the optimization flags?
`-localOptimizations` should take care of that.

I used `-localOptimizations` and the duplicate `null` setting statements were not removed. I also tried with all optimization flags (i.e. `-opt` and `-inline` as well) and have the same result, albeit with shorter names.

> You can also try using the _Jurst_ dialect ...

Thanks for pointing that out. It definitely makes my life much simpler as I don't have to send Wurst a list of files, just the directory that contains the Jass (Jurst) files.
